src/services/comments.js

```diff
@@ -22,10 +25,28 @@ module.exports = class CommentsHelper {
 	 * @param {Integer} resourceId - Resource ID
 	 * @param {Object} bodyData - Request Body
 	 * @param {String} userId - User ID
+	 * @param {String} orgId - Organization ID
 	 * @returns {JSON} - comment id
 	 */
-	static async update(commentId = '', resourceId, bodyData, userId) {
+	static async update(commentId = '', resourceId, bodyData, userId, orgId) {
 		try {
+			//validate resource
+			const resource = await resourceQueries.findOne(
```

Comment:
@priyanka-TL how is this related to activity tracker?

Reply:
Not a part of activity, It was part of validation PR by mistake committed to here, to validate the status of resource